pages/api/notes/index.ts

```diff
@@ -0,0 +1,20 @@
+import type { NextApiRequest, NextApiResponse } from 'next'
+import { PrismaClient } from '@prisma/client'
+import * as client from '../../../lib/api/types';
```

Comment:
```suggestion
import * as client from '@/lib/api/types';
```

Just a personal preference - I'd need to set up the Typescript compiler accordingly of course.